assets/app/view/game/scrap_trains.rb

```diff
@@ -22,27 +23,32 @@ def render
             alignItems: 'center',
           },
         }
-
-        h(:div, [
-          h(:h3, 'Trains to Scrap'),
-          h(:div, div_props, scrap_trains(scrappable_trains)),
-        ])
+        if @game.use_compact_scrap_trains_view
```

Comment:
This choice may vary from step to step. For example, buy_train may want to show the entire scrap train display. Can you add the method to the step instead?

I also recommend calling it something like `scrap_train_button_only`.

Reply:
It's now called from the step, I've added the method to the steps in the games that use it instead of the base step